src/briefcase/platforms/iOS/xcode.py

```diff
@@ -420,9 +416,11 @@ def run_app(self, app: BaseConfig, udid=None, **kwargs):
 
         # Start streaming logs for the app.
         try:
-            print()
-            print("[{app.app_name}] Following simulator log output (type CTRL-C to stop log)...".format(app=app))
-            print("=" * 75)
+            self.logger.info()
+            self.logger.info(
+                "[{app.app_name}] Following simulator log output (type CTRL-C to stop log)...".format(app=app)
+            )
+            self.logger.info("=" * 75)
```

Comment:
Absolutely not in scope for this PR, but worth being aware of if you're at all interested in continuing past this PR: #676 tracks a known issue with capturing log output that we need to address.

Reply:
Thanks for letting me know. I'll take a look.